.travis.yml

```diff
@@ -16,7 +16,6 @@ matrix:
     - env: JOB=HOUSEKEEPING
       python: 3.7
       install:
-        - pip install requests
```

Comment:
Removing **requests** here is fine, it was added recently specifically for the previous version of the script.